test/spec/modules/imuIdSystem_spec.js

```diff
@@ -0,0 +1,43 @@
+import { imuIdSubmodule, storage } from 'modules/imuIdSystem.js';
```

Comment:
Please add additional unit tests. Coverage is currently at only 48%

Reply:
add unit tests.